proposals/instructor-training-selection.md

```diff
@@ -0,0 +1,144 @@
+# Instructor Training Selection
+
+**Warning: This is only a draft.**
+
+**Warning: This only applies for the two-day training course in December 2015.**
+
+## Terminology
+
+-   An **instructor** is someone who teaches a workshop. Instructor training (IT)
+    prepares them to do this.
+
+## Why Select Attends for Instructor Training
+
+SCF/DC wants to train as more instructors as possible
+but demand for instructor training is higher than what SCF could offer
+at the moment.
+
```

Comment:
SCF wants to train as many instructors as possible, but demand for instructor training is higher than what the SCF is able to offer at the moment.